sim/src/pandemic.rs

```diff
@@ -108,7 +148,9 @@ impl PandemicModel {
                             self.transmission(now, person, others, scheduler);
                         }
                     }
-                    _ => {}
+                    _ => {
+                        self.transition(now, person, scheduler);
```

Comment:
Just a note for later, let's not solve it in this PR:
We might be able to use the Scheduler to do this more cleanly. When somebody becomes infected, pick a time in the future when they might recover, and schedule a command to check. When handling that command, ignore it if they've since moved onto a different state. You can also schedule another check in that handler to repeat it. That could be a little more consistent than just doing the check when the person happens to start parking, walking, etc.